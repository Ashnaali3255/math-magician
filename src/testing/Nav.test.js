import React from 'react';
import renderer from 'react-test-renderer';
it('check if the Navbar section renders properly', () => {
const tree = renderer.create(<nav />).toJSON();
  expect(tree).toMatchSnapshot();
});