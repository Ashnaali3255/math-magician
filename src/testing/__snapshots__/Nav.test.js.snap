// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check if the Navbar section renders properly 1`] = `<nav />`;
